README.md

```diff
@@ -52,16 +52,8 @@ sudo apt-get install libldap2-dev libsasl2-dev
 ```bash
 sudo yum install python-devel openldap-devel
 ```
-- Then, for both Debian/Ubuntu and CentOS/RHEL, install `pyOpenSSL` by running:
-```bash
-pip3 install pyOpenSSL
-```
```

Comment:
Good point @QaidVoid and thanks! But can we please keep the `debian/ubuntu` and `centos/rhel` and for `pip3 install pyOpenSSL` instructions and only re-arrange the `requirements.txt` pelase? I prefer this way.

Reply:
Sure. I've made the requested changes.